<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>416</width>
                <height>285</height>
            </rect>
        </property>
        <widget class="QWidget" name="centralWidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                        <item>
                            <widget class="QLabel" name="label_7">
                                <property name="text">
                                    <string>Número de Clientes:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QSpinBox" name="numberOfClientsSpinBox">
                                <property name="minimum">
                                    <number>1</number>
                                </property>
                                <property name="maximum">
                                    <number>1000</number>
                                </property>
                                <property name="value">
                                    <number>1</number>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>IP do Servidor TCP:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="serverIpLineEdit">
                                <property name="text">
                                    <string>127.0.0.1</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                            <widget class="QLabel" name="label_2">
                                <property name="text">
                                    <string>Porta do Servidor TCP:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="serverPortLineEdit">
                                <property name="text">
                                    <string>8080</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                        <item>
                            <widget class="QLabel" name="label_3">
                                <property name="text">
                                    <string>Mensagem TCP:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="tcpMessageLineEdit">
                                <property name="text">
                                    <string>Olá, TCP</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="tcpConnectButton">
                                <property name="text">
                                    <string>Conectar TCP</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="tcpSendButton">
                                <property name="enabled">
                                    <bool>false</bool>
                                </property>
                                <property name="text">
                                    <string>Enviar TCP</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QFrame" name="frame">
                        <property name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                    </widget>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                        <item>
                            <widget class="QLabel" name="label_4">
                                <property name="text">
                                    <string>IP do Servidor UDP:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="udpServerIpLineEdit">
                                <property name="text">
                                    <string>127.0.0.1</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                        <item>
                            <widget class="QLabel" name="label_5">
                                <property name="text">
                                    <string>Porta do Servidor UDP:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="udpServerPortLineEdit">
                                <property name="text">
                                    <string>8081</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                        <item>
                            <widget class="QLabel" name="label_6">
                                <property name="text">
                                    <string>Mensagem UDP:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="udpMessageLineEdit">
                                <property name="text">
                                    <string>Olá, UDP</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="udpConnectButton">
                                <property name="text">
                                    <string>Conectar UDP</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="udpSendButton">
                                <property name="enabled">
                                    <bool>false</bool>
                                </property>
                                <property name="text">
                                    <string>Enviar TCP</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>416</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
